benches/publication/publish.py

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env python
```

Comment:
I don't understand, why you add the file here (it does not look related to the commit) and remove it in the following commit.

Reply:
Just a mistake from copying files from benchs/ to benches/. I think I will squash commits before merging

Reply:
1. I still see those two directories when checked out the top commit.
2. Nope, please, organize commits as they should land into master (before each review iteration). Otherwise the result will be unreadable all-in-one commit.

Reply:
1. The benchs directory is what holds the code for gitlab pipeline (that I do not want to change and break).
2. Ok, I will